### ISSUE TITLE
Conversations are not displayed

### ISSUE THREAD

**Issue by hroncok** [protz/thunderbird-conversations#1061]
I'm running Fedora 23. After I updated to Thunderbird 38.5.0 (unfortunately, I don't know what version I had before, but probably 38.4.0), Conversations stopped working. This is what I see:

![conversations](https://cloud.githubusercontent.com/assets/2401856/13266748/caff4348-da79-11e5-9d1f-e1279618321c.png)

(The names of people in the conversation are clickable and lead to individual messages outside of conversation view (as if the extension is not installed))

This is what I see in debug mode in terminal:

```
2016-02-23 22:01:51	Conversations	DEBUG	Logging enabled
2016-02-23 22:01:51	Conversations.MonkeyPatch	DEBUG	startup, aReason= 1
2016-02-23 22:01:51	Conversations.Modules.Lightning	DEBUG	Did you know, Thunderbird Conversations supports Lightning?
TypeError: nounDef is undefined
-- Exception object --
*
-- Stack Trace --
gloda_ns_newQuery@resource:///modules/gloda/gloda.js:1897:5
ContactIdentityCompleter@resource://gre/components/glautocomp.js:178:22
nsAutoCompleteGloda@resource://gre/components/glautocomp.js:493:26
XPCOMUtils__getFactory/factory.createInstance@resource://gre/modules/XPCOMUtils.jsm:292:19
glodaSearch_XBL_Constructor@chrome://messenger/content/search.xml:75:13
```

When I go to the extension's Preferences and I want to apply the settings things again, it takes more than 2 hours (at this point, I closed the window) and does nothing.

It produces the following debug instantly, but is silent after:

```
2016-02-23 22:10:08	Conversations.AssistantUI	DEBUG	Installing customization actionEnableGloda
2016-02-23 22:10:08	Conversations.AssistantUI	DEBUG	Installing customization actionDontExpand
2016-02-23 22:10:08	Conversations.AssistantUI	DEBUG	Installing customization actionAttachmentsInline
2016-02-23 22:10:08	Conversations.AssistantUI	DEBUG	Installing customization actionOfflineDownload
2016-02-23 22:10:08	Conversations.AssistantUI	DEBUG	Installing customization actionUnifiedInboxSearchesSent
2016-02-23 22:10:08	Conversations.Assistant	DEBUG	[Exception... "Component returned failure code: 0x80004005 (NS_ERROR_FAILURE) [nsIMsgFolder.getChildNamed]"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "JS frame :: resource://conversations/modules/assistant.js :: getSmartFolderNamed :: line 38"  data: no]
2016-02-23 22:10:08	Conversations.Assistant	DEBUG	Is there only one account?
2016-02-23 22:10:08	Conversations.AssistantUI	DEBUG	Installing customization actionSetupView
2016-02-23 22:10:08	Conversations.Assistant	DEBUG	[Exception... "Component returned failure code: 0x80004005 (NS_ERROR_FAILURE) [nsIMsgFolder.getChildNamed]"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "JS frame :: resource://conversations/modules/assistant.js :: getSmartFolderNamed :: line 38"  data: no]
2016-02-23 22:10:08	Conversations.Assistant	DEBUG	Is there only one account?
2016-02-23 22:10:08	Conversations.AssistantUI	ERROR	Error in customization actionSetupView
2016-02-23 22:10:08	Conversations.AssistantUI	ERROR	TypeError: eid(...) is null
2016-02-23 22:10:08	Conversations	DEBUG	
Customizations.actionSetupView.install/moveOn@resource://conversations/modules/assistant.js:213:9
Customizations.actionSetupView.install/waitForIt@resource://conversations/modules/assistant.js:221:11
Customizations.actionSetupView.install@resource://conversations/modules/assistant.js:225:7
install@chrome://conversations/content/assistant/assistant.xhtml:32:31
onFinish@chrome://conversations/content/assistant/assistant.xhtml:83:11
onclick@chrome://conversations/content/assistant/assistant.xhtml:1:1

2016-02-23 22:10:08	Conversations.AssistantUI	DEBUG	Installing customization actionSetupViewDefaults
2016-02-23 22:10:08	Conversations.AssistantUI	DEBUG	Installing customization actionEnsureMessagePaneVisible
```

Also note that my Thunderbird is in English, while stuff from Conversations is in Czech (as is my system), which is really not an issue, it is just weird.

My only other extension is [Awesome ldapInfoShow](https://addons.mozilla.org/en-US/thunderbird/addon/ldapinfoshow/).

**Comment by hroncok** [comment]
BTW Thunderbird Conversations 2.10.2

**Comment by protz** [comment]
Does it help if you install the latest version from 
https://jonathan.protzenko.fr/thunderbird-conversations/?

~ jonathan

**Comment by hroncok** [comment]
Nope, only now everything is in English.

**Comment by rnc** [comment]
Are you sure its not https://bugzilla.redhat.com/show_bug.cgi?id=1311032 ?

What happens if you use a thunderbird binary from mozilla.org instead of Fedora?

**Comment by hroncok** [comment]
Thanks for pointing that out, I never suspected a general problem with my/Fedora's Thunderbird. I haven't tried the Thunderbird binary from mozilla.org, but the proposed fix in bugzilla (38.6.0-3.fc23) works fine.
